lib/generators/boring/rswag/install/install_generator.rb

```diff
@@ -0,0 +1,181 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module Boring
+  module Rswag
+    class InstallGenerator < Rails::Generators::Base
+      desc "Adds rswag gem to the application"
+      source_root File.expand_path("templates", __dir__)
+
+      class_option :rails_port,
+                   type: :string,
+                   desc: "Tell us the port number where you normally run your rails app. Defaults to PORT 3000",
+                   default: "3000"
+      class_option :authentication_type,
+                   type: :string,
+                   desc: "Tell us the authentication type you use in your rails app. Options: ['basic', 'bearer', 'api_key']. Defaults to 'basic'",
+                   default: 'basic'
+      class_option :skip_api_authentication,
+                   type: :boolean,
+                   desc: "Use this option with value 'true' if you don't want to add Authentication when making API calls via swagger docs.",
+                   default: false
+      class_option :api_authentication_options,
+                   type: :string,
+                   desc: 'Use together with authentication_type. Required for "api_key" authentication which has dynamic set of options. See: https://swagger.io/docs/specification/authentication. Example: "{ "name": "api_key", "in": "header" }"'
+      class_option :enable_swagger_ui_authentication,
+                   type: :boolean,
+                   desc: "Use this option with value 'true' for securing your API docs behind a basic authentication to block unauthorized access",
+                   default: false
+      class_option :swagger_ui_authentication_options,
```

Comment:
I would kill this class option and avoid ask some credentials here.

Reply:
Makes perfect sense.

I will remove the option but will still add the configuration required by taking it from Rails credentials. And notify users to update this configuration in after installation message.